### PR TITLE
Topic recognition - ISICs2018 U-Net

### PR DESCRIPTION
**Pull request by salim45018332**
This program aims to solve Task 3 in the report. There are 4 .py modules and 1 main.py as well as a README.md and a folder for plot image. DSC of 0.75 on test data was achieved. I hope you'll accept this pull request. Thanks.

Teguh Salim (45018332)

### CODE REVIEW

**Comment by SiyuLiu0329** [comment]
- Some of the commits were made after deadline
- Sufficiently detailed commit messages
- Code seem okay for Easy Difficulty
Other feedback: you should also include qualitative results as it is a sanity check. Sometimes numbers don't tell everything.

**Comment by rexsw** [comment]
Code looks good for easy difficulty

I have no further feedback good job

File: recognition/45018332-ISICs-UNET/imagegen.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import tensorflow as tf
```
Comment (shakes76) [inline recognition/45018332-ISICs-UNET/imagegen.py:1]: Code header block should be added for all scripts/source files

**Comment by shakes76** [comment]
## TF Usage
OK
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage, Readme sparse
Module present
Images of the Results not provided, minimum quantitative results
## Commit Log
Meaningful commit messages 
Progressive commits used 
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No Request Description. In the future, a more detailed pull request could've been done. When developer receiving the request is not known, the request could be rejected. Remember you could use/reuse the README info.

**Pull request merged**